src/pilot/install-director.sh

```diff
@@ -254,12 +254,6 @@ done
 create_flavor baremetal false
 echo "## Done."
 
```

Comment:
Does this code snipet pertain to deploying Dashboard?

Reply:
not to do with dashboard